lib/promenade/karafka/consumer_subscriber.rb

```diff
@@ -0,0 +1,44 @@
+require "promenade/karafka/subscriber"
+
+module Promenade
+  module Karafka
+    class ConsumerSubscriber < Subscriber
+      attach_to "consumer.karafka"
+
+      Promenade.histogram :kafka_consumer_batch_processing_latency do
+        doc "Consumer message processing latency in milliseconds"
```

Comment:
```suggestion
        doc "Consumer message processing latency in seconds"
```

Best practice is to always use base units ... so for time that is seconds!

Reply:
I have updated, the previous (same) metric, was measured in miliseconds